src/main/java/ft/Main.java

```diff
@@ -0,0 +1,44 @@
+package ft;
+
+import java.io.File;
+import java.nio.file.Files;
+
+// Normaly using * with imports is a bad practice, however in this case it makes experimenting with
+// Fluo easier.
+import org.apache.fluo.api.client.*;
+import org.apache.fluo.api.config.*;
+import org.apache.fluo.api.data.*;
+import org.apache.fluo.api.mini.MiniFluo;
+import org.apache.fluo.api.observer.*;
+
+public class Main {
+  public static void main(String[] args) throws Exception {
+
+    String tmpDir = Files.createTempDirectory(new File("target").toPath(), "mini").toString();
```

Comment:
My comment was that you can replace `new File("target").toPath()` with `Paths.get("target")`